source/Refitting/src/ExtrToSIT.cc

```diff
@@ -191,9 +196,8 @@ void ExtrToSIT::init() {
   // usually a good idea to
   printParameters() ;
   
-  // set up the geometery needed by KalTest
-  //FIXME: for now do KalTest only - make this a steering parameter to use other fitters
-  _trksystem =  MarlinTrk::Factory::createMarlinTrkSystem( "KalTest" , marlin::Global::GEAR , "" ) ;
+  // set up the geometery needed by DDKalTest
+  _trksystem =  MarlinTrk::Factory::createMarlinTrkSystem( _trkSystemName , marlin::Global::GEAR , "" ) ;
   
   if( _trksystem == 0 ){
     
```

Comment:
The exception in the next line should be changed to reflect the name of the trkSystem that couldn't be created